city_scrapers/spiders/wayne_local_emergency_planning.py

```diff
@@ -0,0 +1,99 @@
+from city_scrapers_core.constants import COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil import parser
+
+
+class WayneLocalEmergencyPlanningSpider(CityScrapersSpider):
+    name = "wayne_local_emergency_planning"
+    agency = "Wayne County Local Emergency Planning Committee"
+    timezone = "America/Detroit"
+    allowed_domains = ["www.waynecounty.com"]
+    start_urls = ["https://www.waynecounty.com/departments/hsem/wayne-county-lepc.aspx"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+
+        # the meeting dates I've seen all had the substring 'day, ' as in 'Wednesday, March 6, 2019'
+        meeting_dates = response.xpath('''//p[contains(text(),'day, ')]''')
```

Comment:
This is a good observation, but on occasion we've seen some agencies be inconsistent about including days of the week. It might be safer to get the description text and then search for dates with regex. A pattern we've used a lot has been `[A-Z][a-z]{2,8} \d{1,2},? \d{4}`. The `\d{4}` catches 4 digit numbers and makes it pretty safe to assume there's a year in there.

You can also parse the text content directly without having to manually strip tags, which can be error prone if attributes are added to the elements. The easiest way to do this for XPath is `/text()` or `::text` for CSS selectors and then adding `.extract()` to the selector

Reply:
I've made some changes on my branch that address these. Thanks for the ideas,